docs/03-Contributing/07-API-for-contributors.md

```diff
@@ -1225,3 +1226,45 @@ o
   }
 ]
 ```
+
+### Apply policies spec
+
+Creates and/or modifies the policies included in the specs list. To modify an existing policy, the name of the query included in `specs` must already be used by an existing policy. If a policy with the specified name doesn't exist in Fleet, a new policy will be created.
```

Comment:
Sorry to insist. Let's document that if the policy exists, apply spec will ignore `team` and `platforms` fields (that is, those fields cannot be updated).